src/metabase/driver/sql_jdbc.clj

```diff
@@ -156,6 +158,10 @@
     (doseq [sql sqls]
       (qp.writeback/execute-write-sql! db-id sql))))
 
+(defmethod driver/pk-options :sql-jdbc
+  [_driver]
+  [:auto-increment [:not nil]])
```

Comment:
Why is this auto-increment, and not primary-key as with postgres? These seem like two different things entirely